app/views/admin/product_import/_errors_list.html.haml

```diff
@@ -7,5 +7,3 @@
       ( {{entry.attributes.display_name}} )
   %p.error{ng: {repeat: "(attribute, error) in entry.errors", show: "ignore_fields.indexOf(attribute) < 0" }}
     &nbsp;-&nbsp; {{error}}
-  %p.error{ng: {repeat: "(attribute, error) in entry.product_validations"}}
-    &nbsp;-&nbsp; {{error}}
```

Comment:
I dont understand this change @Matt-Yorkley 
product_validations is still being populated in some cases, why would you remove it from the page?

Reply:
There was some duplication here, as the `product_validations` are actually being added elsewhere. Also the formatting was not right: 

![](https://user-images.githubusercontent.com/6525576/57093343-e2e26100-6d0d-11e9-9c49-48d5ad168cb3.png)

Reply:
ok, I dont understand the code... why do we need to keep both entry.errors and entry.product_validations?

    def mark_as_invalid(entry, options = {})
      entry.errors.add(options[:attribute], options[:error]) if options[:attribute] && options[:error]
      entry.product_validations = options[:product_validations] if options[:product_validations]
    end